gapis/api/vulkan/replay.go

```diff
@@ -977,28 +966,41 @@ func (a API) Replay(
 				}
 			}
 
+			if cfg.drawMode == service.DrawMode_OVERDRAW {
+				// TODO(subcommands): Add subcommand support here
+				if err := earlyTerminator.Add(ctx, extraCommands, api.CmdID(cmdid), req.after[1:]); err != nil {
+					return err
+				}
+
+				if overdraw == nil {
+					overdraw = newStencilOverdraw()
+				}
+				overdraw.add(ctx, uint64(extraCommands), req.after, intent.Capture, rr.Result)
+				break
+			}
+			if err := earlyTerminator.Add(ctx, extraCommands, api.CmdID(cmdid), api.SubCmdIdx{}); err != nil {
+				return err
+			}
+			subIdx := append(api.SubCmdIdx{}, req.after...)
+			subIdx[0] = subIdx[0] + uint64(extraCommands)
+			splitter.Split(ctx, subIdx)
 			switch cfg.drawMode {
 			case service.DrawMode_WIREFRAME_ALL:
 				wire = true
 			case service.DrawMode_WIREFRAME_OVERLAY:
 				return fmt.Errorf("Overlay wireframe view is not currently supported")
-			case service.DrawMode_OVERDRAW:
-				if overdraw == nil {
-					overdraw = newStencilOverdraw()
-				}
-				overdraw.add(ctx, uint64(extraCommands), req.after, intent.Capture, rr.Result)
+			default:
```

Comment:
Maybe add a comment saying that `case service.DrawMode_OVERDRAW` is already handled before this switch, and why?